woo-razorpay.php

```diff
@@ -1456,7 +1470,9 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
 
             $razorpayData = $api->order->fetch($razorpayOrderId);
 
-            $this->updateOrderAddress($razorpayData, $order);
+            $this->newUserAccount($razorpayData, $order);
```

Comment:
Instead of initiating everywhere, can we trigger this function inside `UpdateOrderAddress` function?